lib/index.js

```diff
@@ -256,7 +256,9 @@ Amplitude.prototype.orderCompleted = function(track) {
 
 Amplitude.prototype.group = function(group) {
   var groupId = group.groupId();
-  if (groupId) window.amplitude.setGroup('[Segment] Group', groupId);
+
+  // For advanced reporting, some customers will send a groupType in their properties or traits. Use that if it exists, otherwise default to the old version.
+  if (groupId) window.amplitude.setGroup(group.traits().groupType || '[Segment] Group', groupId);
```

Comment:
my only hesitance is that this is not a spec'd trait for `.group()` calls: https://segment.com/docs/spec/group/

is this `groupType` supposed to be some semantic picklist to enable whatever advance reporting?

But yeah i'd probably run it by the #spec-committee if this should be integration specific option